jenkins-pipelines/manager/debian9-manager-upgrade.jenkinsfile

```diff
@@ -0,0 +1,21 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    params: params,
+    manager: true,
+    backend: 'aws',
+    aws_region: 'us-east-1',
+
+    target_scylla_mgmt_client_repo: '',
+    target_scylla_mgmt_agent_repo: '',
+    scylla_mgmt_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/deb/debian/scylladb-manager-2.0-stretch.list',
+    scylla_mgmt_agent_repo: 'https://repositories.scylladb.com/scylla/repo/qa-test/centos/scylladb-manager-2.0.repo',
```

Comment:
because this is common to all jobs (our scylla + manager agent running always on Centos) shouldn't it be in the "base" pipeline instead of repeating it on every jenkinsfile?

Reply:
Since the regular sanities share the same base pipeline, I'd like to avoid that, since it shouldn't be the default in the other jobs

Reply:
in that case, we could use a "base" yaml for the common entries, and call both yamls for each test (the current yaml + the base yaml).
it will allow us to modify 1 single place every time we need to bump this version